aries_cloudagent/vc/vc_ld/store.py

```diff
@@ -0,0 +1,42 @@
+"""Verifiable Credential storage methods."""
+from ...admin.request_context import AdminRequestContext
+from ...storage.vc_holder.base import VCHolder
+from ...storage.vc_holder.vc_record import VCRecord
+from ...vc.vc_ld import VerifiableCredential
+from ...vc.ld_proofs import DocumentLoader
+from pyld import jsonld
+from pyld.jsonld import JsonLdProcessor
+
+
+async def store_credential(
+    credential: VerifiableCredential,
+    document_loader: DocumentLoader,
+    context: AdminRequestContext,
```

Comment:
Another thing I caught just now, this should take a `Profile` instead of a context of any variety. If this method is moved to the manager, the profile will be provided in the class (`self.profile`). I think this is another good measure of stateful/stateless-ness; the issue, verify, prove modules do not take a context or profile anywhere.